examples/dtls-sock/dtls-client.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       DTLS sock client example
+ *
+ * @author      Aiman Ismail <muhammadaimanbin.ismail@haw-hamburg.de>
+ */
+
+#include <stdio.h>
+
+#include "net/sock/udp.h"
+#include "net/sock/dtls.h"
+#include "net/ipv6/addr.h"
+#include "net/credman.h"
+
+#include "client_keys.h"
+
+#ifndef DTLS_DEFAULT_PORT
+#define DTLS_DEFAULT_PORT 20220 /* DTLS default port */
+#endif
+
+#define SOCK_DTLS_CLIENT_TAG (2)
+
+#ifdef DTLS_ECC
+static ecdsa_public_key_t other_pubkeys[] = {
```

Comment:
This needs to be `static const` now (would make sense anyway, as I don't assume you change this).

Reply:
Fixed. Also fixed in `dtls-server.c`.